ScanbotBarcodeSDKExample/BarcodeSDK.NET.Droid.Example/BarcodeSDK.NET.Droid.Example.csproj

```diff
@@ -10,12 +10,14 @@
         <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
         <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
         <RootNamespace>BarcodeSDK.NET.Droid</RootNamespace>
-        <Platforms>x64;ARM64</Platforms>
+        <Platforms>AnyCPU</Platforms>

```

Comment:
@scan-mayank, AnyCPU shouldn't be here. Especially for Visual Studio for Windows to work correctly, but even more generally since the project is in fact tied to x64 and ARM64.